Version Formatting Fix

cgriggs01 opened the pull request:
The `tf-release-bot` that we use for modifying the changelog requires this specific format that is can interact with. These changes should address this.

mpetazzoni commented on README.md:
```diff
@@ -29,6 +29,14 @@ Changelog is available [here](https://github.com/signalfx/terraform-provider-sig
+
+* `auth_token` (required) - The auth token for [authentication](https://developers.signalfx.com/basics/authentication.html)
+* `api_url` - The API URL to use for communicating with SignalFx. This is helpful for organizations who need to set their Realm or use a proxy.
+* `custom_app_url` - The application URL that users should use to interact with assets in the browser. This is used by organizations with a custom [SSO domain](https://docs.signalfx.com/en/latest/admin-guide/sso.html).
```
Actually anyone that's not in `us0` uses a different app URL, so the main use case here is not necessarily for custom SSO domains. Most likely someone that needs to change `api_url` will be using a different app URL as well.

cory-signalfx replied:
Excellent, thanks!

mpetazzoni commented on CHANGELOG.md:
```diff
@@ -1,5 +1,20 @@
+The following new options were added to the provider's configuration:
+
+* `api_url` which allows users to customize the URL to which API requests will be send. This allows users of other realms or those using proxies to use this provider.
+* `custom_app_url` which allows users to customize the app URL used for viewing resources. This is mostly for users that use SSO with SignalFx, i.e. `https://yourcompany.signalfx.com`.
```
Also need to fix here

cgriggs01 committed:
Version Formatting Fix

The `tf-release-bot` that we use for modifying the changelog requires this specific format that is can interact with. These changes should address this.
